events/events.go

```diff
@@ -171,7 +171,8 @@ type GrenadeEventIf interface {
 // GrenadeEvent contains the common attributes of nade events. Dont register
 // handlers on this tho, you want GrenadeEventIf for that
 type GrenadeEvent struct {
-	GrenadeType     common.EquipmentElement
+	// IMPORTANT: I REMOVED GrenadeType to replace it by Grenade (which give access to its type), this means it could break user's code (feel free to do otherwise)... 
```

Comment:
Bring back `GrenadeType` for compatibility, we can keep both.

Reply:
Sure !

However I don't know if you wish / how to handle deprecation but we might declare `GrenadeType` as deprecated for a futur major version ? (because it's kinda a duplicate field)

Reply:
see https://github.com/markus-wa/demoinfocs-golang/pull/161#discussion_r349000148